docs/release-notes/release-notes-0.14.0.md

```diff
@@ -104,10 +104,10 @@ operations.
 
 ### Explicit Channel Negotiation
 
-[A new protocol extension has been added known as explicit channel negotiation]
-(https://github.com/lightningnetwork/lnd/pull/5669). This allows a channel
-initiator to signal their desired channel type to use with the remote peer. If
-the remote peer supports said channel type and agrees, the previous implicit
+[A new protocol extension has been added known as explicit channel 
+negotiation](https://github.com/lightningnetwork/lnd/pull/5669). This allows a 
+channelinitiator to signal their desired channel type to use with the remote peer.
```

Comment:
not sure but feels like this should be "channel initiator"

Reply:
Ah yes, fixed.